main/zoe/guide/contracts/loan.md

```diff
@@ -0,0 +1,143 @@
+# Loan
+
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts/loan)
+##### [View all contracts on Github](https://github.com/Agoric/agoric-sdk/tree/master/packages/zoe/src/contracts)
+
+The basic loan contract allows a  borrower to add collateral of a
+particular brand and get a loan of another brand. The collateral (also
+known as margin) must be greater than the loan value, at an amount set
+by the Maintenance Margin Requirement (`mmr`) in the terms of the
+contract.
+
+The loan does not have a distinct end time. Rather, if the
+value of the collateral changes such that insufficient margin is
+provided, the collateral is liquidated, and the loan is closed. At any
+time, the borrower can add collateral or repay the loan with interest,
```

Comment:
Might add a parenthetical about where the interest rate and compounding period are specified.

Reply:
Hmm, I think that is explained enough below